handlers.go

```diff
@@ -309,6 +309,16 @@ func CombinedLoggingHandler(out io.Writer, h http.Handler) http.Handler {
 // See http://httpd.apache.org/docs/2.2/logs.html#common for a description of this format.
 //
 // LoggingHandler always sets the ident field of the log to -
+//
+// Example:
+//
+//  r := mux.NewRouter()
+//  r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+//  	w.Write([]byte("This is a catch-all route"))
+//  })
+//  logAndServe := handlers.LoggingHandler(os.Stdout, r)
```

Comment:
Thanks for the naming idea. This example actually came from one of my tutorials on github.com/AndyHaskell/MEAN-Gopher where it's named logAndServe there, so if I give you a shoutout for the name there, mind if I update the name of that handler to loggedRouter in my own tutorial for more accuracy?